react/features/whiteboard/constants.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Fixed name of the whiteboard fake participant.
+ */
+export const WHITEBOARD_PARTICIPANT_NAME = 'Whiteboard';
+
+export const WHITEBOARD = 'whiteboard';
+
+export const WHITEBOARD_STORAGE_KEY = 'excalidraw-collab';
```

Comment:
Let's put "jitsi" in there to avoid any potential collisions. Or do we need that exact key name so excalidraw reads it?

Reply:
👍  Please add a comment.